.gitlab-ci.yml

```diff
@@ -13,54 +13,49 @@ variables:
   S3_BUCKET_PATH: "mender-cli"
   BUILD_DIR: build
 
-before_script:
-  - mkdir -p /go/src/$(dirname $REPO_NAME)/mender-cli /go/src/_/builds
-  - cp -r $CI_PROJECT_DIR /go/src/$(dirname $REPO_NAME)
-  - cd /go/src/$(dirname $REPO_NAME)/mender-cli
-  - apt-get update && apt-get install -yyq liblzma-dev
-
 stages:
-  - test_prep
-  - test_fast
   - test
+  - test_acceptance_build
+  - test_acceptance
   - build
   - publish
 
-test:build_acceptance:tools:
+test_acceptance_build:tools:
   image: golang:1.14
-  stage: test_prep
+  stage: test_acceptance_build
   script:
-    - go build -o $CI_PROJECT_DIR/mender-cli
-    - chmod +x $CI_PROJECT_DIR/mender-cli
-    - wget -q -O $CI_PROJECT_DIR/mender-artifact https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/master/mender-artifact
-    - chmod +x $CI_PROJECT_DIR/mender-artifact
+    - apt-get update && apt-get install make
+    # make target env var
+    - export PROJECT_DIR="$CI_PROJECT_DIR/"
+    - make build-acceptance-tools
   artifacts:
     untracked: true
     paths:
       - mender-cli
       - mender-artifact
 
-test:build_acceptance:image:
-  stage: test_prep
+test_acceptance_build:image:
+  stage: test_acceptance_build
   image: docker
   services:
     - docker:19.03.5-dind
   script:
-    - docker build -t testing -f tests/Dockerfile .
+    - apk add make
+    - export PROJECT_DIR="$CI_PROJECT_DIR/"
+    - make build-acceptance-image
     - docker save testing > $CI_PROJECT_DIR/acceptance_testing_image.tar
   artifacts:
     expire_in: 2w
     paths:
       - acceptance_testing_image.tar
 
-test:format:
+test:static:
   image: golang:1.14
   stage: test
-  allow_failure: true
+  allow_failure: false
```

Comment:
This can be completely removed; `false` is the default so better to clear it up.